dev/oranda.json

```diff
@@ -1,6 +1,7 @@
 {
   "homepage": "https://oranda.axo.dev",
   "readme_path": "dev/README.md",
+  "theme": "light",
```

Comment:
Would you mind defaulting to black?

Reply:
So the reason I did this is to make sure when people are testing dev stuff, they don't forget to test the theme option and they don't forget to consider light mode. I would prefer not to change it, but I am open to reasons I haven't considered.

Reply:
oh this is the dev yeah sorry